device-types/Ciena/waveserver-ai.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: Ciena
+model: Waveserver Ai
+slug: ciena-waveserver-ai
+part_number: 186-1010-900
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
```

Comment:
Generally you do not need `subdevice_role` unless the device either slots into another (in which case it would be a child), or has `device-bays` to house child devices. If this device does not need this, please remove it